Monika After Story/game/screens.rpy

```diff
@@ -426,7 +426,7 @@ style input:
 ## http://www.renpy.org/doc/html/screen_special.html#choice
 
 screen choice(items):
-    style_prefix ("choice" if not mas_globals.dark_mode else "choice_dark")
+    style_prefix ("choice" + mas_ui.ui_mode_suffix)
```

Comment:
concatenation still has a cost, its best to just have var strings for this instead of trying to build it everytime. We need to optimize as much as we can given the perf issues with our version of renpy.